components/Card/style.css

```diff
@@ -0,0 +1,146 @@
+.card{
+    background: var(--white-color);
+    border: 1px solid black;
+    padding: 10px;
+    min-width: 300px;
+    min-height: 200px;
```

Comment:
try to use relative units instead of absolute units. You can use px for <10px

Reply:
Yeah thank you :)